Mapping/ndt_map/ndt_map.py

```diff
@@ -52,14 +52,15 @@ def __init__(self, ox, oy, resolution):
         center_y = np.mean(oy)
         self.ox = ox
         self.oy = oy
+        # init grid_map
+        self.grid_map = GridMap(width, height, resolution, center_x, center_y, self.NDTGrid())
```

Comment:
This change is harmless I think. But, could you please let me know, why this change is needed? What kind of issue happened in your environment?